src/parquet_writer.h

```diff
@@ -0,0 +1,278 @@
+#ifndef PARQUET_WRITER_H
+#define PARQUET_WRITER_H
+
+#include <napi.h>
+
+#include <parquet/arrow/writer.h>
+#include <parquet/stream_writer.h>
+
+#include <vector>
+
+#include "field_type.h"
+
+parquet::ConvertedType::type ConvertedTypeFromFieldType(FieldType::Type type) {
```

Comment:
`static`, to mark them as file-local functions

Reply:
Huh, I never stopped to realize that you could use static outside class member declarations and variables inside function bodies.

Reply:
Yes, it's super bad that it's used for such different use cases but it works that way.